src/utils/sendResult.js

```diff
@@ -0,0 +1,9 @@
+import HttpStatus from 'http-status-codes';
+
+const sendResult = async (res, status, msg, data) => {
+  if (data && data.length === 0) {
+    return res.status(status).json({ status: HttpStatus.NOT_FOUND, msg: 'Oops no results found' });
+  }
+  return res.status(status).json({ status: HttpStatus.OK, msg, data });
```

Comment:
@victorkarangwa4 remember to use customize helper class to return all successful and error responses.

Reply:
Thanks @NiyoEric for the feedback
This function acts like a customizer